talks/abstracts/CollabW2022.md

```diff
@@ -0,0 +1,78 @@
+# SSI Collaborations Workshop 2020 - Mini Workshop
+
+- **Where:** Online! :woman_technologist:
+- **When:** 4th - 7th April 2022
+- **Conference website:** <https://software.ac.uk/cw22>
+
+---
+
+## Name / Email
+
+Sarah Gibson / drsarahlgibson@gmail.com
+
+## Home Institution
+
+The International Interactive Computing Collaboration (2i2c)
+
+## Session Collaborators
+
+_Please include First Name, Surname, Home institution and email address for any additional collaborators on this session proposal, each on a separate line._
+
+- Meag Doherty, NIH
+- Min Ragan-Kelley, Simula
+- Achintya Rao, The Alan Turing Institute
+
+## Mini-workshop or demo session title
+
+Crowdsourcing Community Practices for Defining Reproducible Computational Environments in the Cloud
+
+## Mini-workshop or demo session abstract
+
+_If your session is accepted this abstract will be made public on the CW22 website, so please write this as a public description containing 200-400 words._
+
+_Current word count: 230_
+
+mybinder.org is a very popular tool for hosting interactive and reproducible computational environments in the cloud.
```

Comment:
Maybe make the first line a bit punchier?

```markdown
mybinder.org makes your reproducible notebooks interactive via the cloud.
The ethos of this popular service for hosting interactive computational environments is…
```

Reply:
I generally like it, I'm only going to push back on explicitly calling out notebooks. We have enough trouble telling people you can use scripts and a terminal in binder as it is 😉  So something like this maybe?

```
mybinder.org transforms your static code into a reproducible and interactive environment running in the cloud.
```